tutorials/src/bin/basic-tutorial-5.rs

```diff
@@ -0,0 +1,377 @@
+#[cfg(feature = "tutorial5")]
```

Comment:
I'd suggest to move all this into a sub-module so that you have to put the feature gate only once or twice :)

Reply:
Done